BUILD.bazel

```diff
@@ -472,41 +472,254 @@ ray_cc_library(
 
 ray_cc_library(
     name = "plasma_store_server_lib",
+    srcs = ["src/ray/object_manager/plasma/store_runner.cc"],
+    hdrs = ["src/ray/object_manager/plasma/store_runner.h"],
+    copts = PLASMA_COPTS,
+    linkopts = PLASMA_LINKOPTS,
+    deps = [
+        ":object_lifecycle_manager",
+        ":object_store",
+        ":plasma_allocator",
+        ":store",
+        "//src/ray/common:asio",
+        "//src/ray/common:file_system_monitor",
+        "//src/ray/common:ray_config",
+        "//src/ray/util:thread_utils",
+        "@com_google_absl//absl/synchronization",
+    ],
+)
+
+ray_cc_library(
+    name = "store",
+    srcs = ["src/ray/object_manager/plasma/store.cc"],
+    hdrs = ["src/ray/object_manager/plasma/store.h"],
+    deps = [
+        ":connection_protocol",
+        ":create_request_queue",
+        ":eviction_policy",
+        ":get_request_queue",
+        ":malloc",
+        ":object_lifecycle_manager",
+        ":object_manager_common",
+        ":object_manager_plasma_common",
+        ":object_store",
+        ":plasma_allocator",
+        ":stats_metric",
+        "//src/ray/common:asio",
+        "//src/ray/common:file_system_monitor",
+        "//src/ray/common:id",
+        "//src/ray/common:network",
+        "//src/ray/common:ray_config",
+        "//src/ray/common:status",
+        "//src/ray/util",
+        "@boost//:bind",
+        "@com_google_absl//absl/base:core_headers",
+        "@com_google_absl//absl/container:flat_hash_set",
+        "@com_google_absl//absl/synchronization",
+    ],
+)
+
+ray_cc_library(
+    name = "get_request_queue",
+    srcs = ["src/ray/object_manager/plasma/get_request_queue.cc"],
+    hdrs = ["src/ray/object_manager/plasma/get_request_queue.h"],
+    deps = [
+        ":connection_protocol",
+        ":object_lifecycle_manager",
+        "//src/ray/common:asio",
+        "//src/ray/common:id",
+    ],
+)
+
+ray_cc_library(
+    name = "object_lifecycle_manager",
+    srcs = ["src/ray/object_manager/plasma/object_lifecycle_manager.cc"],
+    hdrs = ["src/ray/object_manager/plasma/object_lifecycle_manager.h"],
+    deps = [
+        ":connection_protocol",
+        ":eviction_policy",
+        ":object_manager_common",
+        ":object_store",
+        ":plasma_allocator",
+        ":stats_collector",
+        "//src/ray/common:asio",
+        "//src/ray/common:id",
+        "//src/ray/common:ray_config",
+        "@com_google_absl//absl/container:flat_hash_set",
+        "@com_google_absl//absl/time",
+        "@com_google_googletest//:gtest",
+    ],
+)
+
+ray_cc_library(
+    name = "stats_collector",
+    srcs = ["src/ray/object_manager/plasma/stats_collector.cc"],
+    hdrs = ["src/ray/object_manager/plasma/stats_collector.h"],
+    deps = [
+        ":object_manager_plasma_common",
+        ":stats_metric",
+        "//src/ray/util:counter_map",
+    ],
+)
+
+ray_cc_library(
+    name = "malloc",
```

Comment:
please prefix with `plasma`, `malloc` is too general (this is a root build file)